config/routes.rb

```diff
@@ -11,12 +11,6 @@
   resources :named_contacts, only: :create
 
   namespace :anonymous_feedback do
-    get 'problem_reports', to: redirect {|p, req| req.params[:path] ? "/anonymous_feedback?path=" + req.params[:path] : "/anonymous_feedback"}
```

Comment:
Kibana log search for this endpoint:
<img width="632" alt="screen shot 2016-09-13 at 14 16 37" src="https://cloud.githubusercontent.com/assets/1370570/18717192/14ccda88-8017-11e6-9433-437e9a356682.png">